.github/workflows/build.yml

```diff
@@ -248,17 +256,19 @@ jobs:
             --mount "type=bind,src=$PWD/signed,dst=/doi-build/signed"
 
             # https://explore.ggcr.dev/?repo=docker/image-signer-verifier
-            docker/image-signer-verifier:0.3.3@sha256:a5351e6495596429bacea85fbf8f41a77ce7237c26c74fd7c3b94c3e6d409c82
+            "$BASHBREW_IMAGE_SIGNER"
 
             sign
```

Comment:
we will need docker credentials for this step to push to `$BASHBREW_REFERRERS_REPO`

Reply:
I moved the login from the push command to its separate step